pkgs/tools/misc/sd-mux-ctrl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, pkgconfig, libftdi1, popt}:
+
+stdenv.mkDerivation rec {
+  pname = "sd-mux-ctrl";
+  version = "2020-02-17";
+
+  src = fetchgit {
+    url = "https://git.tizen.org/cgit/tools/testlab/sd-mux.git";
+    rev = "9dd189d973da64e033a0c5c2adb3d94b23153d94";
+    sha256 = "0fxl8m1zkkyxkc2zi8930m0njfgnd04a22acny6vljnzag2shjvg";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
+  buildInputs = [ libftdi1 popt ];
+
+  postInstall = ''
+    install -D -m 644 ../doc/man/${pname}.1 $out/share/man/man1/${pname}.1
```

Comment:
Because of the previous change for `pname`, this will need to be changed. Which might not be much of an issue since it's anyway more correct (I think) to use the full name of the file rather than assuming that `pname` actually matches.

Reply:
Definitely makes sense